src/Test/Factories.php

```diff
@@ -29,13 +30,15 @@ public static function _setUpFactories(): void
 
         TestState::bootFromContainer($kernel->getContainer());
         Factory::configuration()->setManagerRegistry(
-            new LazyManagerRegistry(static function() {
-                if (!static::$booted) {
-                    static::bootKernel();
-                }
+            new LazyManagerRegistry(
+                static function () {
+                    if (!static::$booted) {
+                        static::bootKernel();
+                    }
 
-                return static::$kernel->getContainer()->get('doctrine');
-            })
+                    return static::$kernel->getContainer()->get(ChainManagerRegistry::class);
```

Comment:
I think we're going to have a problem here with non-bundle usage: `ChainManagerRegistry` isn't in the container. I think we'll have to do what we do in the compiler pass manually here - maybe as a helper method on TestState to avoid duplication?